workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/message/PinMessageExecutor.java

```diff
@@ -0,0 +1,45 @@
+package com.symphony.bdk.workflow.engine.executor.message;
+
+import com.symphony.bdk.core.util.IdUtil;
+import com.symphony.bdk.gen.api.model.V1IMAttributes;
+import com.symphony.bdk.gen.api.model.V3RoomAttributes;
+import com.symphony.bdk.gen.api.model.V4Message;
+import com.symphony.bdk.workflow.engine.executor.ActivityExecutor;
+import com.symphony.bdk.workflow.engine.executor.ActivityExecutorContext;
+import com.symphony.bdk.workflow.swadl.v1.activity.message.PinMessage;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+
+@Slf4j
+public class PinMessageExecutor implements ActivityExecutor<PinMessage> {
+
+  public static final String IM = "IM";
+  public static final String ROOM = "ROOM";
```

Comment:
Can't we reuse the generated `com.symphony.bdk.gen.api.model.StreamType` class here?

Reply:
yes sure, unfortunately I cannot use directly the enum since one check needs a string and the other one needs V2StreamType, but at least I can use them to create the constants. wdyt? https://github.com/finos/symphony-wdk/pull/59/commits/b2c33101303fe841022b6d6186cd4a383d3cda2c#diff-645a9462ddbf3c495bde99004204750223bca0102c4fe3d8e652f403efe1c12cR19